Common/Data/Custom/SmartInsider/SmartInsiderIntention.cs

```diff
@@ -209,9 +209,8 @@ public override BaseData Reader(SubscriptionDataConfig config, string line, Date
             {
                 Symbol = config.Symbol
             };
-            // Files are made available at the earliest @ 17:00 U.K. time
-            intention.Time = intention.Time.AddHours(17).ConvertTo(TimeZones.London,config.DataTimeZone);
 
+            intention.Time = intention.Time.ConvertFromUtc(config.DataTimeZone);
```

Comment:
Think these conversions (here and in https://github.com/QuantConnect/Lean/pull/4439/files#diff-fac74792ac2784b4b039c950f38c2fa2R241) could be avoided if SmartInsider specified UTC as data time zone? Default is `TimeZones.NewYork`

Reply:
Good point, I'll update the PR 👍